maintainers/maintainer-list.nix

```diff
@@ -9638,6 +9638,11 @@
     githubId = 1102396;
     name = "Jussi Maki";
   };
+  joanmassachs = {
+    github = "joanmassachs";
+    githubId = 22916782;
+    name = "Joan Massachs";
```

Comment:
Are you sure you don't want to add an email? This is not enforced for now but would be good to provide alternative contact

Reply:
Yeah, if it is not enforced, I prefer not to add it. I don't think it will be necessary for maintaining just this small package :)